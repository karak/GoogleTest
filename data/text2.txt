cxlomvtp

cxlomvtp
phkvsmflf
jwpalfp
cniktbylkj
eroftc
xkgmpe
tkecngx
fdopbjt
psbce
jwwmgfx
clfufbqsat
eohtfimof
xnisiudgp
tbrplwsnj
fvbhkmvedec
pmdglcfkx
jkzet
cvrwpvitf
erowhnp
xzjybj
twygrcwfc
fgoze