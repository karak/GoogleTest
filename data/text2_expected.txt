cxlomvtp
phkvsmflf
fdopbjt
tkecngx
xkgmpe
eohtfimof
fvbhkmvedec
clfufbqsat
twygrcwfc
cvrwpvitf
fgoze
eroftc
cniktbylkj
jkzet
tbrplwsnj
jwpalfp
psbce
erowhnp
pmdglcfkx
xnisiudgp